include/fastdds/rtps/writer/ReaderLocator.h

```diff
@@ -53,7 +54,8 @@ class ReaderLocator : public RTPSMessageSenderInterface
      * @param max_multicast_locators  Maximum number of multicast locators to hold.
      */
     ReaderLocator(
-            RTPSParticipantImpl* owner,
+            RTPSParticipantImpl* participant_owner,
```

Comment:
We could just pass the `RTPSWriter*` and use `owner->getRTPSParticipant()` to initialize `participant_owner_`

The Doxydoc should be updated to reflect this change.